docker-compose.yml

```diff
@@ -0,0 +1,107 @@
+version: "3.7"
+
+x-logging:
+  &default-logging
+  driver: "json-file"
+  options:
+    max-size: "2m"
+    max-file: "2"
+
+services:
+  portainer:
+    image: portainer/portainer-ce:2.1.1-alpine
+    hostname: portainer
+    container_name: portainer
+    restart: unless-stopped
+    # Admin User password: Password01
+    volumes:
+      # Linux container under linux:
+        #- /var/run/docker.sock:/var/run/docker.sock
+        #- ~/code/DataPersistent/portainer_data:/data
+      # Linux container under windows:
+      - /var/run/docker.sock:/var/run/docker.sock
+      - ./DockerVolumes/portainer_data:/data
+      #- D:\Temp:/data
+      # Windows container:
+      # Windows container:
+        #- \\.\pipe\docker_engine:\\.\pipe\docker_engine
+        #-C:\ProgramData\Portainer:C:\data 
+    ports:
+      - 8000:8000
+      - 9000:9000
+    networks:
+      - docker-network
+    logging: *default-logging      
+
+  keycloak_SA:
```

Comment:
why not just "keycloak" ?

Reply:
As it is standalone and does not required any other containers to run.